requirements.txt

```diff
@@ -13,5 +13,8 @@ tldextract
 # Used locally by trustymail code even when Lambda is used
 dnspython
 
+# Used by some scanners to parse pages
+beautifulsoup4
```

Comment:
I believe that the dependencies for scanners are in the `requirements-scanners.txt` file, so I believe we should add it there -- but TBH I'm not sure why the requirements are architected this way. 

Also, any reason not to pin the version?

Reply:
Update: looks like it already is in requirements-scanners.txt